README.md

```diff
@@ -21,10 +27,14 @@ A list of amazing Newsletters.
 
 ### CSS
 * [CSS Weekly](http://css-weekly.com/)
+* [CSS-Tricks](https://css-tricks.com) signup is at the bottom of the page.
```

Comment:
Yes, they have the premium service, but at the bottom of the page is a signup for the free newsletter. I'll try to find a direct link, but so far I haven't seen one.